enhancements/update/update-blocker-lifecycle/README.md

```diff
@@ -0,0 +1,190 @@
+* example: Admin uses oc to fix things
+* example: Admin must SSH to hosts, restore from backups, or other non standard admin activities
+
+Is this a regression (if all previous versions were also vulnerable, updating to the new, vulnerable version does not increase exposure)?
+* example: No, it has always been like this we just never noticed
+* example: Yes, from 4.y.z to 4.y+1.z Or 4.y.z to 4.y.z+1
+
+### User Stories
+
+#### A developer wondering about a serious bug
+
+Before this enhancement, the "is this an update blocker?" process was less discoverable.
+With this enhancement, the concerned developer only needs to add the `UpgradeBlocker` keyword to initiate the process.
+And they also have access to this document to more easily understand the rest of the process, if they need to push the whole decision through before an update monitor is available to help out.
+
+#### A component maintainer assigned to a bug
+
+This enhancement adds labels to make it clear whether the bug assignee is currently responsible for providing an impact statement (`ImpactStatementRequested`), or whether the bug assignee has fulfilled their responsibility and moved the bug to `ImpactStatementProposed`.
+
+#### An update monitor managing multiple bugs
+
+This enhancement formalizes the various steps in the decision process, allowing for some steps to be automated, and giving a clear `ImpactStatementProposed` queue for final block-or-not considertion.
+
+### API Extensions
+
+No API; just internal process.
+
+### Risks and Mitigations
+
+No risks.
+
+## Design Details
+
+It's all up in [the *proposal* section](#proposal).
+
+### Test Plan
+
+No test plan.
+
+### Graduation Criteria
+
+No graduation criteria; this is internal policy and has no backwards-compatibility commitments.
+
+#### Dev Preview -> Tech Preview
+
+No graduation criteria.
+
+#### Tech Preview -> GA
+
+No graduation criteria.
+
+#### Removing a deprecated feature
+
+We might decide to do something completely different tomorrow.
+Don't build anything on top of this process that you would be sad about throwing away.
+
+### Upgrade / Downgrade Strategy
+
+This enhancement is only intended to help ongoing graph-data operation.
+If we pivot strategies, we will likely abandon any closed bugs without porting them to the new strategy.
+Or we may port closed bugs to new strategies, if that makes implementing the next strategy easier.
+
+### Version Skew Strategy
+
+All of the data is in Bugzilla, and we control all the consumers for this internal workflow.
+So if we pivot strategies, we can turn off any robots and port everything that needs porting at once.
+There is no need to provision for version skew.
+
+### Operational Aspects of API Extensions
+
+No API; just internal process.
+
+#### Failure Modes
+
+No API; just internal process.
+
+#### Support Procedures
+
+No API; just internal process.
+
+## Implementation History
+
+No implementation.
+
+## Drawbacks
+
+No drawbacks.
+
+## Alternatives
```

Comment:
was there a reason we couldn't use flags (UpgradeBlocker?, UpgradeBlocker+, UpgradeBlocker-) instead of keywords?

UpgradeBlocker? would equate to "impact statement requested"

Reply:
(flags would be more consistent with our release blocking mechanism)